web-client/src/presenter/presenter.ts

```diff
@@ -860,41 +869,62 @@ export const presenterSequences = {
   gotoMessageDetailSequence: gotoMessageDetailSequence as unknown as Function,
   gotoMessagesSequence: gotoMessagesSequence as unknown as Function,
   gotoMyAccountSequence: gotoMyAccountSequence as unknown as Function,
+
   gotoPdfPreviewSequence: gotoPdfPreviewSequence as unknown as Function,
+
   gotoPendingReportSequence: gotoPendingReportSequence as unknown as Function,
+
```

Comment:
Are these empty lines intentional?

Reply:
I'm not sure. @akuny @pixiwyn should we remove these?

Reply:
I'm going to remove them. If anyone has strong feelings about this, feel free to undo!